support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -640,55 +674,132 @@ function CheckoutComponent({ geoId }: Props) {
 						>
 							<Box cssOverrides={shorterBoxMargin}>
 								<BoxContents>
-									<PersonalDetails
-										email={email}
-										firstName={firstName}
-										lastName={lastName}
-										isSignedIn={isSignedIn}
-										// TODO - ONE_OFF support, this should be true when ONE_OFF
-										hideNameFields={false}
-										onEmailChange={(email) => {
-											setEmail(email);
-										}}
-										onFirstNameChange={(firstName) => {
-											setFirstName(firstName);
-										}}
-										onLastNameChange={(lastName) => {
-											setLastName(lastName);
-										}}
-										errors={{}}
-										signOutLink={<Signout isSignedIn={isSignedIn} />}
-										contributionState={
-											showStateSelect && (
-												<StateSelect
-													countryId={countryId}
-													state={'STATE'}
-													onStateChange={() => {
-														//  no-op
+									<div css={personalDetailsFieldGroupStyles(true)}>
```

Comment:
@jamesgorrie regarding the stricture of the form, and your [comment](https://github.com/guardian/support-frontend/pull/5982/files#r1593733388) above regarding fieldsets, is there a reason we use a `<div>` to wrap the personal details and not a `<fieldset>` element?

Reply:
As we're pushing to use as many built-in features of the standard HTML form on this page a review of the HTML seems like a good idea - should we create a card to review the HTML as suggested?

Reply:
https://trello.com/c/KYYngPNf/840-make-generic-checkout-use-standard-dom-form-fields